features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/attachments/preview/AttachmentsPreviewPresenter.kt

```diff
@@ -240,7 +251,6 @@ class AttachmentsPreviewPresenter @AssistedInject constructor(
             cleanUp(mediaUploadInfo)
             // Reset the sendActionState to ensure that dialog is closed before the screen
             sendActionState.value = SendActionState.Done
-            onDoneListener()
```

Comment:
We still need to invoke this in the case of `sendOnBackground` is false, i.e. `FeatureFlags.MediaUploadOnSendQueue` is disabled.

Confirmed by test: the screen does not close